src/main/java/bank/account/AccountRepository.java

```diff
@@ -0,0 +1,15 @@
+package bank.account;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface AccountRepository extends JpaRepository<Account, Long> {
+    Optional<Account> findByUid(String uid);
+
+    Optional<Account> findByUidAndPersonid(String uid, Long person_id);
```

Comment:
```suggestion
    Optional<Account> findByUidAndPerson(String uid, Person person);
```
Тут також будуть зміни